runtime/js/98_global_scope.js

```diff
@@ -157,51 +158,51 @@ class Navigator {
 
 const navigator = webidl.createBranded(Navigator);
 
-let numCpus, userAgent, language;
-
-function setNumCpus(val) {
-  numCpus = val;
-}
-
-function setUserAgent(val) {
-  userAgent = val;
+function makeLazy(f) {
```

Comment:
Nit: to be slightly clearer about the memoization part of this function, can we use the common name for this operation (maybe `memoize` or `memoizeLazy`)?